### DISCUSSION TITLE
problemas de acceso desde los notebooks

### DISCUSSION THREAD

**Discussion by rramosp** [rramosp/ai4eng.v1#117] [00 - General]
algunos de vosotros estáis teniendo problemas para autenticaros a la plataforma de correcciones desde google colab.

Esto sólo afecta a los que regularmente usan el botón Login with Google desde los notebooks y es debido a un cambio que hizo Google en el procedimiento técnico de autenticación.

Mientras entendemos bien el problema para solucionarlo, debéis de entrar siempre con login + password (no con el botón Login with Google)

Si no lo habéis hecho nunca (y no tenéis un password), tenéis que:

1. entrar a la plataforma y pinchar sobre change/reset pwd para establecer un password nuevo (recibiréis un correo con instrucciones).
2. usar siempre la opción Login with email, para entrar desde los notebooks en colab.

como se indica en las imágenes a continuación
saludos
raúl

![image](https://user-images.githubusercontent.com/13835425/161086894-6c481f73-722e-4668-a6fc-755990dfca05.png)

![image](https://user-images.githubusercontent.com/13835425/161086923-83438c34-4ea6-491e-b1ff-2968471d1a03.png)

**Comment by joanfco30** [comment]
Muchas gracias por la información. Problema solucionado. 

![image](https://user-images.githubusercontent.com/60493083/161102755-82cc8d2c-7cfd-4155-b536-da19d5d149f1.png)

**Comment by ebm7pc** [comment]
Buenas tardes. Yo ya tenía suscripciones al curso en semestres anteriores. Podrían anular las suscripciones anteriores? Tengo problemas con en login en los notebooks de colab por esta razón. Mi correo es:
ariel.bedoya@udea.edu.co

Muchas gracias!

**Reply by Felipe-RA** [reply]
Anotado!

Estaremos avisando cuando ya este listo

**Comment by Jhon2507** [comment]
Buenas noches, me aparecen dos cursos en la plataforma y desde el colab no me deja avanzar la entrada. Mi correo es: jhon.longas1@udea.edu.co

**Reply by Felipe-RA** [reply]
Saludos Jhon

Ya lo pongo en el post apropiado
